src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -145,6 +145,9 @@
         </StringProperty.DataSource>
     </StringProperty>
     <BoolProperty Name="PackageRequireLicenseAcceptance" Default="False" Visible="False"/>
+    <StringProperty Name="PackageLicenseFileLocation" Visible="False"/>
```

Comment:
Is this needed? I don't see any usages.

Reply:
No, that was left over from the pack target implementation.